src/networkAddresses.ts

```diff
@@ -26,6 +26,7 @@ export const ORACLE_BNB = "0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB";
 export const NETWORK_ADDRESSES = {
   bscmainnet: {
     DEFAULT_PROPOSER_ADDRESS: "0x97a32D4506F6A35De68e0680859cDF41D077a9a9",
+    ACCESS_CONTROL_MANAGER: "0x4788629ABc6cFCA10F9f969efdEAa1cF70c23555",
```

Comment:
Why don't we use the address from the npm package in this case (as we are doing for bsctestnet)? Anyway, I'm not sure if should add to this file the ACM address, because the framework doesn't use it. We could simply define this address in the VIP's where we need it. I think that is the general rule

Reply:
> Why don't we use the address from the npm package in this case

Address casing is incorrect in the governance repo (it's all lowercase there), I'll make a PR to change that

> I'm not sure if should add to this file the ACM address, because the framework doesn't use it

I think the framework shoud use it, it's one of our main contracts and its address is unlikely to change in future. The direction we seem to be going with the framework is to offload more responsibilities to it (e.g. it already checks risk params and market configuration). It could also check ownership / access controls at some point. I can add the corresponding checks (i.e. move https://github.com/VenusProtocol/vips/blob/3a001ba52b1e10f63f80c665fc2b37e0c351f66d/simulations/vip-400/bsctestnet.ts#L50-L56 to the framework).

> I think that is the general rule

Unrelated to all of the above, I think we should discuss this further. It's true that at some point we decided to use hardcoded addresses in VIPs. However, IMO, with more and more networks and deployments the PR reviews become quite time consuming and error-prone – we need to manually check every address, and the more addresses we have hardcoded, the easier it is to miss a mistake in one of them. I think there should be a better approach here (maybe something along the lines of what I mentioned [in this thread](https://github.com/VenusProtocol/oracle/pull/228#discussion_r1784503796)). If we could check the deployment __commit hash__ rather than all the addresses during the reviews, this would simplify things and make errors less probable.